packages/middleware-ssec/src/index.ts

```diff
@@ -59,6 +65,16 @@ export function ssecMiddleware(options: PreviouslyResolved): InitializeMiddlewar
     };
 }
 
+function base64ToUint8Array(base64String: string) {
+  const binaryString = atob(base64String);
```

Comment:
test failing with ReferenceError: atob is not defined

Reply:
Added logic to check if Buffer is available (Nodejs), and if not use atob which is supported on browsers.